src/game/scenes/mainmenu/menu_connect.c

```diff
@@ -17,11 +17,26 @@ typedef struct {
     int controllers_created;
     ENetHost *host;
     component *addr_input;
+    component *port_input;
     component *connect_button;
     component *cancel_button;
     scene *s;
 } connect_menu_data;
 
+enum
+{
+    lowest_port = 1,
```

Comment:
we should probably start from 1025, since the first 1024 ports are still considered "special".